runmariadb.sh

```diff
@@ -28,6 +28,7 @@ EOSQL
     # mysqld_safe closes stdout/stderr if no bash options are set ($- == '')
     # turn on tracing to prevent this
     exec bash -x /usr/bin/mysqld_safe --init-file /tmp/configure-mysql.sql
+    rm /tmp/configure-mysql.sql
```

Comment:
Hm, won't this file continue to exist until mysqld stops? Maybe something like this works? 

```bash
exec bash -x /usr/bin/mysqld_safe --init-file <(<<EOF
DELETE FROM mysql.user ;
CREATE USER 'ironic'@'localhost' identified by '${MARIADB_PASSWORD}' ;
GRANT ALL on *.* TO 'ironic'@'localhost' WITH GRANT OPTION ;
DROP DATABASE IF EXISTS test ;
CREATE DATABASE IF NOT EXISTS  ironic ;
FLUSH PRIVILEGES ;
EOF
)
```

Reply:
Right! I tried your suggestion but it seems there was some problem, thus I pushed a new version that removes the config file once the process has been started

Reply:
> One more thing: could we get this in upstream first? It would benefit as well.
> 
> https://github.com/metal3-io/ironic-image/blob/master/runmariadb.sh

Opened new pr upstream https://github.com/metal3-io/ironic-image/pull/224